package.json

```diff
@@ -51,10 +62,14 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.2.1",
     "ethereumjs-wallet": "^1.0.1",
-    "jest": "^27.0.6",
+    "jest": "^29.0.0",
     "prettier": "^2.8.1",
     "prettier-plugin-packagejson": "^2.3.0",
-    "sinon": "^11.1.1"
+    "sinon": "15.0.1",
+    "ts-jest": "^29.0.3",
+    "ts-node": "^10.7.0",
+    "typedoc": "^0.23.15",
+    "typescript": "~4.8.4"
```

Comment:
nb: `4.9.3-5` stable vers also out and `5.0.0` stable due [today](https://github.com/microsoft/TypeScript/issues/51362)
but if problematic can always upgrade to these later